    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185032.json', OSError(28, 'No space left on device'))
2024-07-26 18:50:32,560:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-07-26 18:50:32,560:INFO:run_script: running scripts/010-deploy.sh
2024-07-26 18:50:32,567:ERROR:run_script: scripts/010-deploy.sh:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:50:32,567:INFO:run_script: completed scripts/010-deploy.sh
[pid: 837474|app: 0|req: 3381/3379] 140.82.115.160 () {54 vars in 914 bytes} [Fri Jul 26 18:50:32 2024] POST /ud_hook => generated 2 bytes in 10 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185302.json', OSError(28, 'No space left on device'))
2024-07-26 18:53:02,087:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:53:02,094:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:53:02,094:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3382/3380] 140.82.115.26 () {54 vars in 913 bytes} [Fri Jul 26 18:53:02 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185320.json', OSError(28, 'No space left on device'))
2024-07-26 18:53:20,860:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-07-26 18:53:20,860:INFO:run_script: running scripts/010-deploy.sh
2024-07-26 18:53:20,868:ERROR:run_script: scripts/010-deploy.sh:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:53:20,868:INFO:run_script: completed scripts/010-deploy.sh
[pid: 837474|app: 0|req: 3383/3381] 140.82.115.252 () {54 vars in 914 bytes} [Fri Jul 26 18:53:20 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185416.json', OSError(28, 'No space left on device'))
2024-07-26 18:54:16,756:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:54:16,763:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:54:16,764:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3384/3382] 140.82.115.62 () {54 vars in 913 bytes} [Fri Jul 26 18:54:16 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185456.json', OSError(28, 'No space left on device'))
2024-07-26 18:54:56,455:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:54:56,462:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:54:56,462:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3385/3383] 140.82.115.26 () {54 vars in 913 bytes} [Fri Jul 26 18:54:56 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185532.json', OSError(28, 'No space left on device'))
2024-07-26 18:55:32,912:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:55:32,920:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:55:32,920:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3386/3384] 140.82.115.44 () {54 vars in 913 bytes} [Fri Jul 26 18:55:32 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185541.json', OSError(28, 'No space left on device'))
2024-07-26 18:55:41,834:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:55:41,841:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:55:41,841:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3387/3385] 140.82.115.29 () {54 vars in 913 bytes} [Fri Jul 26 18:55:41 2024] POST /ud_hook => generated 2 bytes in 10 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185604.json', OSError(28, 'No space left on device'))
2024-07-26 18:56:05,453:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:56:05,467:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:56:05,467:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3388/3386] 140.82.115.11 () {54 vars in 913 bytes} [Fri Jul 26 18:56:04 2024] POST /ud_hook => generated 2 bytes in 749 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185716.json', OSError(28, 'No space left on device'))
2024-07-26 18:57:16,212:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:57:16,219:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:57:16,219:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3389/3387] 140.82.115.168 () {54 vars in 914 bytes} [Fri Jul 26 18:57:16 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726185813.json', OSError(28, 'No space left on device'))
2024-07-26 18:58:13,831:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 18:58:13,839:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 18:58:13,839:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3390/3388] 140.82.115.248 () {54 vars in 914 bytes} [Fri Jul 26 18:58:13 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726190019.json', OSError(28, 'No space left on device'))
2024-07-26 19:00:19,043:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-07-26 19:00:19,043:INFO:run_script: running scripts/010-deploy.sh
2024-07-26 19:00:19,050:ERROR:run_script: scripts/010-deploy.sh:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 19:00:19,051:INFO:run_script: completed scripts/010-deploy.sh
[pid: 837474|app: 0|req: 3391/3389] 140.82.115.250 () {54 vars in 914 bytes} [Fri Jul 26 19:00:19 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240726190058.json', OSError(28, 'No space left on device'))
2024-07-26 19:00:59,026:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-26 19:00:59,039:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-26 19:00:59,039:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3392/3390] 140.82.115.252 () {54 vars in 914 bytes} [Fri Jul 26 19:00:58 2024] POST /ud_hook => generated 2 bytes in 947 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240727234021.json', OSError(28, 'No space left on device'))
2024-07-27 23:40:22,684:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-27 23:40:22,691:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-27 23:40:22,692:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3393/3391] 140.82.115.26 () {54 vars in 913 bytes} [Sat Jul 27 23:40:21 2024] POST /ud_hook => generated 2 bytes in 1022 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240727234056.json', OSError(28, 'No space left on device'))
2024-07-27 23:40:56,735:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-27 23:40:56,747:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-27 23:40:56,748:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3394/3392] 140.82.115.246 () {54 vars in 914 bytes} [Sat Jul 27 23:40:56 2024] POST /ud_hook => generated 2 bytes in 18 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240727234135.json', OSError(28, 'No space left on device'))
2024-07-27 23:41:42,702:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-27 23:41:42,710:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-27 23:41:42,711:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3395/3393] 140.82.115.94 () {54 vars in 913 bytes} [Sat Jul 27 23:41:35 2024] POST /ud_hook => generated 2 bytes in 7456 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240727234206.json', OSError(28, 'No space left on device'))
2024-07-27 23:42:06,639:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-07-27 23:42:06,653:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-07-27 23:42:06,653:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3396/3394] 140.82.115.62 () {54 vars in 913 bytes} [Sat Jul 27 23:42:06 2024] POST /ud_hook => generated 2 bytes in 19 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-04 23:58:10,447:INFO:event: Ignoring push on branch refs/heads/pashto_docs
[pid: 837474|app: 0|req: 3397/3395] 140.82.115.253 () {54 vars in 914 bytes} [Sun Aug  4 23:58:10 2024] POST /ud_hook => generated 2 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-05 00:53:06,223:INFO:event: Ignoring push on branch refs/heads/pashto_docs
[pid: 837474|app: 0|req: 3398/3396] 140.82.115.243 () {54 vars in 914 bytes} [Mon Aug  5 00:53:06 2024] POST /ud_hook => generated 2 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-05 00:56:45,332:INFO:event: Ignoring push on branch refs/heads/pashto_docs
[pid: 837474|app: 0|req: 3399/3397] 140.82.115.73 () {54 vars in 913 bytes} [Mon Aug  5 00:56:45 2024] POST /ud_hook => generated 2 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-05 01:10:25,428:INFO:event: Ignoring push on branch refs/heads/pashto_docs
[pid: 837474|app: 0|req: 3400/3398] 140.82.115.43 () {54 vars in 913 bytes} [Mon Aug  5 01:10:25 2024] POST /ud_hook => generated 2 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-05 01:25:33,948:INFO:event: Ignoring push on branch refs/heads/pashto_docs
[pid: 837474|app: 0|req: 3401/3399] 140.82.115.57 () {54 vars in 913 bytes} [Mon Aug  5 01:25:33 2024] POST /ud_hook => generated 2 bytes in 1 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-07 01:09:28,042:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-07 01:18:37,824:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating 99f4e1f326..690dd4eb69\nFast-forward\n _bej/dep/appos.md                          |   2 +-\n _config.yml                                |   2 +\n _cop/dep/{obl-npmod.md => obl-unmarked.md} |  18 ++---\n _de/feat-index.md                          |  11 +++\n _de/feat/ExtPos.md                         |  93 ++++++++++++++++++++++++++\n _el/dep/compound.md                        |   2 +-\n _en/dep/nmod-npmod.md                      |   4 +-\n _en/dep/nmod-tmod.md                       |   3 +\n _en/dep/obl-npmod.md                       |   3 +\n _en/dep/obl-tmod.md                        |   3 +\n _he/dep/nmod-npmod.md                      |   2 +-\n _he/dep/nmod-tmod.md                       |   4 +-\n _he/dep/nmod-unmarked.md                   |  19 ++++++\n _he/dep/obl-npmod.md                       |   2 +-\n _he/dep/obl-tmod.md                        |   2 +-\n _he/dep/obl-unmarked.md                    |  19 ++++++\n _he/index.md                               |   5 +-\n _includes/at_glance_sapling.html           |  82 ++++++++++++++++++++++-\n _qpm/index.md                              |  20 +++---\n {_uz => _qxp}/template-index.md            |  10 +--\n _tr/index.md                               |  10 +--\n _tr/tokenization.md                        |   6 +-\n _uz/dep/advcl.md                           |   7 ++\n _uz/dep/advmod.md                          |   7 ++\n _uz/dep/ccomp.md                           |  16 +++++\n _uz/dep/csubj-part.md                      |  13 +++-\n _uz/dep/csubj.md                           |   7 ++\n _uz/dep/nmod-part.md                       |   7 ++\n _uz/dep/nsubj.md                           |   7 ++\n _uz/dep/obj.md                             |   7 ++\n _uz/dep/obl.md                             |   7 ++\n _uz/feat/Case.md                           |  76 +--------------------\n _uz/feat/Degree.md                         |  54 +++++++++++++++\n _uz/feat/NumType.md                        |  70 ++++++++++++++++++++\n _uz/feat/PronType.md                       | 101 ++++++++++++++++++++++++++++\n _uz/index.md                               | 103 +++++++++++++++++++++++++++++\n 36 files changed, 680 insertions(+), 124 deletions(-)\n rename _cop/dep/{obl-npmod.md => obl-unmarked.md} (56%)\n create mode 100644 _de/feat-index.md\n create mode 100644 _de/feat/ExtPos.md\n create mode 100644 _he/dep/nmod-unmarked.md\n create mode 100644 _he/dep/obl-unmarked.md\n rename {_uz => _qxp}/template-index.md (83%)\n create mode 100644 _uz/feat/Degree.md\n create mode 100644 _uz/feat/NumType.md\n create mode 100644 _uz/feat/PronType.md\n create mode 100644 _uz/index.md\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-07 01:18:37,824:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   99f4e1f326..690dd4eb69  pages-source -> origin/pages-source\n * [new branch]            pashto_docs  -> origin/pashto_docs\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-07 01:18:37,824:INFO:run_script: completed scripts/010-deploy.sh --incremental
Wed Aug  7 01:18:37 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.172) !!!
Wed Aug  7 01:18:37 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.172)
OSError: write error
[pid: 837474|app: 0|req: 3402/3400] 140.82.115.172 () {54 vars in 914 bytes} [Wed Aug  7 01:09:28 2024] POST /ud_hook => generated 0 bytes in 549787 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
2024-08-21 16:22:30,754:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-21 16:22:30,801:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-21 16:27:30,386:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating 690dd4eb69..29b0259801\nFast-forward\n _config.yml            |  2 ++\n _kbc/template-index.md | 59 ++++++++++++++++++++++++++++++++++++++++++++++++++\n 2 files changed, 61 insertions(+)\n create mode 100644 _kbc/template-index.md\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-21 16:27:30,386:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   690dd4eb69..29b0259801  pages-source -> origin/pages-source\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-21 16:27:30,386:INFO:run_script: completed scripts/010-deploy.sh --incremental
Wed Aug 21 16:27:30 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.160) !!!
Wed Aug 21 16:27:30 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.160)
OSError: write error
[pid: 837474|app: 0|req: 3403/3401] 140.82.115.160 () {54 vars in 914 bytes} [Wed Aug 21 16:22:30 2024] POST /ud_hook => generated 0 bytes in 299776 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
2024-08-21 21:33:51,446:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-21 21:33:51,447:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-21 21:34:13,945:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating 29b0259801..b5577c8450\nFast-forward\n _ab/dep/advcl-purp.md |  2 +-\n _ab/dep/conj-q.md     | 21 +++++++++++++++++++++\n 2 files changed, 22 insertions(+), 1 deletion(-)\n create mode 100644 _ab/dep/conj-q.md\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-21 21:34:13,946:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   29b0259801..b5577c8450  pages-source -> origin/pages-source\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-21 21:34:13,946:INFO:run_script: completed scripts/010-deploy.sh --incremental
Wed Aug 21 21:34:13 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.91) !!!
Wed Aug 21 21:34:13 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.91)
OSError: write error
[pid: 837474|app: 0|req: 3404/3402] 140.82.115.91 () {54 vars in 915 bytes} [Wed Aug 21 21:33:51 2024] POST /ud_hook => generated 0 bytes in 22527 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
2024-08-21 22:23:55,353:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-21 22:23:55,353:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-21 22:24:16,223:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating b5577c8450..ead39bf19c\nFast-forward\n _ab/dep/xcomp-obj.md  |  2 +-\n _ab/dep/xcomp-subj.md | 20 ++++++++++++++++++++\n 2 files changed, 21 insertions(+), 1 deletion(-)\n create mode 100644 _ab/dep/xcomp-subj.md\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-21 22:24:16,223:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   b5577c8450..ead39bf19c  pages-source -> origin/pages-source\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-21 22:24:16,223:INFO:run_script: completed scripts/010-deploy.sh --incremental
Wed Aug 21 22:24:16 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.79) !!!
Wed Aug 21 22:24:16 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.79)
OSError: write error
[pid: 837474|app: 0|req: 3405/3403] 140.82.115.79 () {54 vars in 913 bytes} [Wed Aug 21 22:23:55 2024] POST /ud_hook => generated 0 bytes in 20901 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
2024-08-24 18:01:27,624:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-24 18:01:27,822:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-24 18:04:13,849:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating ead39bf19c..b5bf76a568\nFast-forward\n _includes/at_glance_sapling.html | 112 ++++++++++++++++++++++++++++++++++++++-\n _includes/lang_spec_docs.html    |   1 +\n 2 files changed, 112 insertions(+), 1 deletion(-)\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-24 18:04:13,850:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   ead39bf19c..b5bf76a568  pages-source -> origin/pages-source\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-24 18:04:13,850:INFO:run_script: completed scripts/010-deploy.sh --incremental
Sat Aug 24 18:04:13 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.45) !!!
Sat Aug 24 18:04:13 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.45)
OSError: write error
[pid: 837474|app: 0|req: 3406/3404] 140.82.115.45 () {54 vars in 913 bytes} [Sat Aug 24 18:01:27 2024] POST /ud_hook => generated 0 bytes in 166615 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825172537.json', OSError(28, 'No space left on device'))
2024-08-25 17:25:37,187:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 17:25:37,235:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 17:25:37,235:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3407/3405] 140.82.115.60 () {54 vars in 913 bytes} [Sun Aug 25 17:25:37 2024] POST /ud_hook => generated 2 bytes in 172 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825174817.json', OSError(28, 'No space left on device'))
2024-08-25 17:48:17,474:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 17:48:17,485:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 17:48:17,485:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3408/3406] 140.82.115.250 () {54 vars in 914 bytes} [Sun Aug 25 17:48:17 2024] POST /ud_hook => generated 2 bytes in 16 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825175849.json', OSError(28, 'No space left on device'))
2024-08-25 17:58:49,912:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-25 17:58:49,912:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 17:58:49,919:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 17:58:49,919:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3409/3407] 140.82.115.62 () {54 vars in 913 bytes} [Sun Aug 25 17:58:49 2024] POST /ud_hook => generated 2 bytes in 9 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825183238.json', OSError(28, 'No space left on device'))
2024-08-25 18:32:38,454:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-25 18:32:38,454:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 18:32:38,467:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 18:32:38,468:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3410/3408] 140.82.115.161 () {54 vars in 914 bytes} [Sun Aug 25 18:32:38 2024] POST /ud_hook => generated 2 bytes in 18 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825184421.json', OSError(28, 'No space left on device'))
2024-08-25 18:44:21,380:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 18:44:21,394:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 18:44:21,395:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3411/3409] 140.82.115.242 () {54 vars in 914 bytes} [Sun Aug 25 18:44:21 2024] POST /ud_hook => generated 2 bytes in 20 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825210714.json', OSError(28, 'No space left on device'))
2024-08-25 21:07:14,134:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-25 21:07:14,135:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 21:07:14,148:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 21:07:14,148:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3412/3410] 140.82.115.250 () {54 vars in 914 bytes} [Sun Aug 25 21:07:14 2024] POST /ud_hook => generated 2 bytes in 19 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825211513.json', OSError(28, 'No space left on device'))
2024-08-25 21:15:13,111:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 21:15:13,123:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 21:15:13,123:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3413/3411] 140.82.115.46 () {54 vars in 913 bytes} [Sun Aug 25 21:15:13 2024] POST /ud_hook => generated 2 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240825213921.json', OSError(28, 'No space left on device'))
2024-08-25 21:39:21,270:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-25 21:39:21,278:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-25 21:39:21,278:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3414/3412] 140.82.115.31 () {54 vars in 913 bytes} [Sun Aug 25 21:39:21 2024] POST /ud_hook => generated 2 bytes in 11 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
--- Logging error ---
Traceback (most recent call last):
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 53, in log_event
    f.write(s)
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/__init__.py", line 1085, in emit
    msg = self.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 929, in format
    return fmt.format(record)
  File "/usr/lib/python3.8/logging/__init__.py", line 668, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.8/logging/__init__.py", line 373, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ginter/.local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 139, in event
    fn = log_event(pretty_print_json(data))
  File "/home/ginter/UD_DOCS_HOOK/jekyll-hook/jekyllhook.py", line 55, in log_event
    logging.error('failed to write {}: {}', fn, e)
Message: 'failed to write {}: {}'
Arguments: ('events/20240826151059.json', OSError(28, 'No space left on device'))
2024-08-26 15:10:59,199:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-26 15:10:59,230:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b"mktemp: failed to create directory via template \xe2\x80\x98./tmp.XXXXXXXXXX\xe2\x80\x99: No space left on device\nrealpath: missing operand\nTry 'realpath --help' for more information.\n"
2024-08-26 15:10:59,230:INFO:run_script: completed scripts/010-deploy.sh --incremental
[pid: 837474|app: 0|req: 3415/3413] 140.82.115.84 () {54 vars in 913 bytes} [Mon Aug 26 15:10:59 2024] POST /ud_hook => generated 2 bytes in 36 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
2024-08-27 15:13:43,650:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-27 15:21:12,395:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating b5bf76a568..17e8280b8b\nFast-forward\n _bgs/template-index.md           |  59 ++++++++\n _config.yml                      |   6 +\n _drs/template-index.md           |  59 ++++++++\n _el/dep/compound-redup.md        |   2 +-\n _includes/at_glance_sapling.html | 304 ++++++++++++++++++++++++++++++++++++++-\n _ssp/template-index.md           |  59 ++++++++\n 6 files changed, 487 insertions(+), 2 deletions(-)\n create mode 100644 _bgs/template-index.md\n create mode 100644 _drs/template-index.md\n create mode 100644 _ssp/template-index.md\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-27 15:21:12,396:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   b5bf76a568..17e8280b8b  pages-source -> origin/pages-source\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-27 15:21:12,396:INFO:run_script: completed scripts/010-deploy.sh --incremental
Tue Aug 27 15:21:12 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.144) !!!
Tue Aug 27 15:21:12 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.144)
OSError: write error
[pid: 837474|app: 0|req: 3416/3414] 140.82.115.144 () {54 vars in 914 bytes} [Tue Aug 27 15:13:43 2024] POST /ud_hook => generated 0 bytes in 448750 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
2024-08-28 14:38:46,804:INFO:event: Detected added/removed files, will run all scripts with --full-rebuild
2024-08-28 14:38:46,804:INFO:run_script: running scripts/010-deploy.sh
2024-08-28 14:39:28,422:INFO:run_script: scripts/010-deploy.sh:OUT: b'Updating 17e8280b8b..ab790f0f6d\nFast-forward\n _ssp/index.md | 45 +++++++++++++++++++++++++++++++++++++++++++++\n 1 file changed, 45 insertions(+)\n create mode 100644 _ssp/index.md\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: disabled. Enable with --incremental\n      Generating... \n'
2024-08-28 14:39:28,422:ERROR:run_script: scripts/010-deploy.sh:ERR: b'From github.com:UniversalDependencies/docs\n   17e8280b8b..ab790f0f6d  pages-source -> origin/pages-source\n\x1b[31m             Error: YAML Exception reading /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_tt/dep/appos.md: (<unknown>): did not find expected key while parsing a block mapping at line 2 column 1\x1b[0m\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-28 14:39:28,422:INFO:run_script: completed scripts/010-deploy.sh
Wed Aug 28 14:39:28 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.168) !!!
Wed Aug 28 14:39:28 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.168)
OSError: write error
[pid: 837474|app: 0|req: 3417/3415] 140.82.115.168 () {54 vars in 914 bytes} [Wed Aug 28 14:38:46 2024] POST /ud_hook => generated 0 bytes in 41623 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
2024-08-28 16:43:39,323:INFO:run_script: running scripts/010-deploy.sh --incremental
2024-08-28 16:43:58,005:INFO:run_script: scripts/010-deploy.sh --incremental:OUT: b'Updating ab790f0f6d..2c1d4287cd\nFast-forward\n _tt/dep/appos.md | 2 +-\n 1 file changed, 1 insertion(+), 1 deletion(-)\nConfiguration file: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_config.yml\n            Source: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source\n       Destination: /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_site\n Incremental build: enabled\n      Generating... \n'
2024-08-28 16:43:58,005:ERROR:run_script: scripts/010-deploy.sh --incremental:ERR: b'From github.com:UniversalDependencies/docs\n   ab790f0f6d..2c1d4287cd  pages-source -> origin/pages-source\n\x1b[31m  Liquid Exception: Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_de/feat-index.md\x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n\x1b[31m      Jekyll 4.1.1   Please append `--trace` to the `build` command \x1b[0m\n\x1b[31m                     for any additional information or backtrace. \x1b[0m\n\x1b[31m                    ------------------------------------------------\x1b[0m\n/home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:223:in `locate_include_file\': Could not locate the included file \'de-feat-table.html\' in any of ["/home/ginter/UD_DOCS_HOOK/jekyll-hook/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. (IOError)\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/tags/include.rb:205:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/liquid_renderer/file.rb:36:in `render!\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:127:in `render_liquid\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:80:in `render_document\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/renderer.rb:63:in `run\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:532:in `render_regenerated\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:517:in `block (2 levels) in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:516:in `block in render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `each_value\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:515:in `render_docs\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:207:in `render\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/site.rb:80:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:28:in `process_site\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:65:in `build\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:36:in `process\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `each\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/command.rb:91:in `process_with_graceful_fail\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go\'\n\tfrom /home/ginter/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program\'\n\tfrom /home/ginter/gems/gems/jekyll-4.1.1/exe/jekyll:15:in `<top (required)>\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `load\'\n\tfrom /home/ginter/gems/bin/jekyll:23:in `<main>\'\n'
2024-08-28 16:43:58,005:INFO:run_script: completed scripts/010-deploy.sh --incremental
Wed Aug 28 16:43:58 2024 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /ud_hook (ip 140.82.115.172) !!!
Wed Aug 28 16:43:58 2024 - uwsgi_response_writev_headers_and_body_do(): Broken pipe [core/writer.c line 306] during POST /ud_hook (140.82.115.172)
OSError: write error
[pid: 837474|app: 0|req: 3418/3416] 140.82.115.172 () {54 vars in 914 bytes} [Wed Aug 28 16:43:39 2024] POST /ud_hook => generated 0 bytes in 18687 msecs (HTTP/1.1 200) 2 headers in 0 bytes (0 switches on core 0)
